wordpress-mobile/WordPress-iOS
# Fix/16941 - Close account change error flow

This PR changes alert shown in case user cannot close account due to having atomic site to a more generic one, with a **Contact Support** button.
This change is needed because having an atomic site is something that's considered being an implementation detail user might not necessarily know about.

## To test:
1. Make sure the current user has an atomic site
2. Under **My Site** tab tap on profile button at the top right corner
3. Tap on **Account Settings**
4. Tap on **Close account**
5. Close account alert is displayed
6. Tap on **Contact Support**
7. New alert is displayed for entering contact info
8. Upon entering contact info user can leave a message to support center

## Flow
Atomic site error | Enter contact info dialog | Contact us
--- | --- | ---
![Simulator Screen Shot - iPhone 11 - 2021-08-24 at 15 44 03](https://user-images.githubusercontent.com/12729242/130627613-b1aae3b4-a4ab-496c-b4aa-917cb4ecb483.png) | ![IMG_2BE159ACAD98-1](https://user-images.githubusercontent.com/12729242/130536320-0cf412d8-1a0e-461c-870a-2017417bbf1f.jpeg) | ![Simulator Screen Shot - iPhone 11 - 2021-08-24 at 01 42 12](https://user-images.githubusercontent.com/12729242/130536200-2d448afb-b67d-40df-9701-56d791b032f1.png)


## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
